packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLines.js

```diff
@@ -17,24 +18,47 @@ export const ConsensusLine = styled('g')`
   }
 `
 
-function TranscribedLines ({ lines, scale, task, theme }) {
-  function createMark (line) {
-    const { activeTool, activeToolIndex, setActiveMark } = task
+function TranscribedLines ({ lines, onFinish, scale, task, theme }) {
+  const ref = createRef()
```

Comment:
Hm... not sure if that would be a problem with refs or with how the state of the input was being managed at the time. `useRef` and `createRef` are pretty different functionally under the hood, and I've had issues with the hooks version not being available immediately for use in the component... but the ref in this case isn't used until an event is triggered, so that might be a moot point.

Reply:
If this works, then I'm inclined to leave it alone. I've run into issues where `createRef().current` is null and doesn't update without a change of state or props.